streamingcli/project/build_command.py

```diff
@@ -55,6 +55,10 @@ def get_jars(converted_notebook: ConvertedNotebook,
                  local_project_config: LocalProjectConfig,
                  notebook_dir: str) -> None:
         jar_handler = JarHandler(project_root_dir=os.getcwd())
+        plugin_jars_paths = jar_handler.get_jars_using_plugin().split(";")
```

Comment:
We are acquiring a classpath and then immediately split it. Maybe `get_jars_using_plugin` should return a `List[str]` instead?

Reply:
Yes, I had the same thoughts. But we are using already classpath in streaming jupyter integrations :/ Shall I change it here and there or leave it?